src/Orleans.SDK.targets

```diff
@@ -9,9 +9,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <!-- Work out the path to ClientGenerator.exe -->
   <Choose>
     <!-- Using Orleans NuGet packages -->
-    <When Condition="Exists('$(MSBuildThisFileDirectory)..\tools\ClientGenerator.exe')">
+    <When Condition="Exists('$(MSBuildThisFileDirectory)../tools/ClientGenerator.exe')">
       <PropertyGroup>
-        <CodeGenToolExeDir>$(MSBuildThisFileDirectory)..\tools</CodeGenToolExeDir>
+        <CodeGenToolExeDir>$(MSBuildThisFileDirectory)../tools</CodeGenToolExeDir>
       </PropertyGroup>
     </When>
     <!-- Internal build within Orleans.sln -->
```

Comment:
in the future we should just get rid of this logic from this file that we ship in the nuget package. We can just have this pre-set in the BuildDirectory.props and that's it